policy-report/falco-adapter/README.md

```diff
@@ -0,0 +1,58 @@
+# Falco adapter
+The falco adapter runs as an output for falcosidekick when enabled, the adapter receives falco events (Falco ships with a default set of rules that check the kernel for unusual behavior) and produces N+1 reports (1 cluster-wide policy report and N namespace specific reports) based on the [Policy Report Custom Resource Definition](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report)
```

Comment:
suggested updates:

The Falco Policy Report adapter receives Falco events and produces one or more [Policy Reports]([Policy Report Custom Resource Definition]). It runs as an optional output in (Falcosidekick)[https://github.com/falcosecurity/falcosidekick]. The adapter produces N+1 reports - 1 cluster-wide policy report and N namespace specific reports.

Reply:
Hey @JimBugwadia , I have made these changes in the latest commit.
Thank you!